go.mod

```diff
@@ -38,6 +38,7 @@ require (
 	github.com/kr/text v0.2.0 // indirect
 	github.com/moby/term v0.0.0-20201110203204-bea5bbe245bf
 	github.com/mrunalp/fileutils v0.0.0-20171103030105-7d4729fb3618
+	github.com/nxadm/tail v1.4.6 // indirect
```

Comment:
I think you can remove that line and run a `go mod tidy`.

Reply:
Nice catch, thanks!